cli/script/command-executor.ts

```diff
@@ -1180,48 +939,12 @@ function isBinaryOrZip(path: string): boolean {
         || path.search(/\.ipa$/i) !== -1;
 }
 
-function throwForMissingCredentials(accessKeyName: string, providerName: string, providerUniqueId: string): void {
-    if (!accessKeyName) throw new Error("Access key is missing.");
-    if (!providerName) throw new Error("Provider name is missing.");
-    if (!providerUniqueId) throw new Error("Provider unique ID is missing.");
-}
-
-function throwForInvalidAccessKeyId(accessKeyId: string, accessKeyName: string): void {
-    if (!accessKeyId) {
-        throw new Error("Access key \"" + accessKeyName + "\" does not exist.");
-    }
-}
-
-function throwForInvalidApp(app: App, appName: string): void {
```

Comment:
I will need to add this as part of a server change - currently we just get: `[Error]  Not Found`